instrumentation/apache-camel-2.20/README.md

```diff
@@ -0,0 +1,5 @@
+# Settings for the Apache Camel instrumentation
+
+| System property | Type | Default | Description |
+|---|---|---|---|
+| `otel.instrumentation.apache-camel.experimental-span-attributes` | Boolean | `false` | Enable the capture of span attributes (experimental).  |
```

Comment:
Should it be
```suggestion
| `otel.instrumentation.apache-camel.experimental-span-attributes` | Boolean | `false` | Enable the capture of experimental span attributes.  |
```
?

Otherwise the current wording sounds like the support of capturing ALL span attributes is experimental and has to be enabled. What this option actually does is enabling attributes, which are not defined by Otel semantic convention and which we call "experimental".

Reply:
Ah, I feel for that one. Well spotted! Fixed now using your suggestion (thanks!).